autoinstrumentation/python/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10-alpine AS build
+
+WORKDIR /operator-build
+
+ADD requirements.txt .
+
+RUN mkdir workspace && pip install --target workspace -r requirements.txt
+
+FROM busybox
```

Comment:
I was wondering why we are using `busybox` as a base image. Other operator images are based on `gcr.io/distroless/static:nonroot`. If there is no reason I would prefer to be consistent.

Reply:
The distroless image doesn't contain the `cp` command which we need.